cfgov/unprocessed/js/organisms/EmailSignup.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+// Required modules.
+var atomicHelpers = require( '../modules/util/atomic-helpers' );
+var ERROR_MESSAGES = require( '../config/error-messages-config' );
+var FORM_MESSAGES = ERROR_MESSAGES.FORM.SUBMISSION;
+var Notification = require( '../molecules/Notification' );
+
+/**
+ * EmailSignup
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the organism.
+ * @returns {EmailSignup} An instance.
+ */
+function EmailSignup( element ) {
```

Comment:
It seems like some of this  (xhr, serialize, etc) could be generalized or moved into utility functions, but that could be done if/when we have another applicable use case.